seqr/models.py

```diff
@@ -397,115 +397,68 @@ class Sample(ModelWithGUID):
         # ('ILLUMINA_INFINIUM_250K', ),
     )
 
-    sample_type = models.CharField(max_length=3, choices=SAMPLE_TYPE_CHOICES)
-
-    individual = models.ForeignKey('Individual', on_delete=models.PROTECT, null=True)
-
-    # This sample_id should be used for looking up this sample in the underlying dataset (for
-    # example, for variant callsets, it should be the VCF sample id). It is not a ForeignKey
-    # into another table.
-    sample_id = models.TextField(db_index=True)
-
-    # biological sample status
-    sample_status = models.CharField(max_length=1, choices=SAMPLE_STATUS_CHOICES, default='S')
-
-    #funding_source = models.CharField(max_length=20, null=True)
-
-    is_external_data = models.BooleanField(default=False)
-
-
-    #sample_batch = models.ForeignKey('SampleBatch', on_delete=models.PROTECT, null=True)
-
-    # reference back to xbrowse base_project is a temporary work-around to support merging of
-    # different projects into one - including those that contain different types of callsets
-    # such as exome and genome
-    deprecated_base_project = models.ForeignKey('base.Project', null=True, on_delete=models.SET_NULL)
-
-    def __unicode__(self):
-        return self.sample_id.strip()
-
-    def _compute_guid(self):
-        return 'S%06d_%s' % (self.id, _slugify(str(self)))
-
-    class Meta:
-       # unique_together = ('sample_batch', 'sample_id')
-       json_fields = [
-           'guid', 'created_date', 'sample_type', 'sample_id', 'sample_status',
-       ]
-
-
-class Dataset(ModelWithGUID):
-    """Dataset represents a single analysis type and result file"""
-
-    ANALYSIS_TYPE_ALIGNMENT = 'ALIGN'
-    ANALYSIS_TYPE_VARIANT_CALLS = 'VARIANTS'
-    ANALYSIS_TYPE_SV = 'SV'
-    ANALYSIS_TYPE_BREAKPOINTS = 'BREAK'
-    ANALYSIS_TYPE_SPLICE_JUNCTIONS = 'SPLICE'
-    ANALYSIS_TYPE_ASE = 'ASE'
-    ANALYSIS_TYPE_CHOICES = (
-        (ANALYSIS_TYPE_ALIGNMENT, 'Alignment'),
-        (ANALYSIS_TYPE_VARIANT_CALLS, 'Variant Calls'),
-        (ANALYSIS_TYPE_SV, 'SV Calls'),
-        (ANALYSIS_TYPE_BREAKPOINTS, 'Breakpoints'),
-        (ANALYSIS_TYPE_SPLICE_JUNCTIONS, 'Splice Junction Calls'),
-        (ANALYSIS_TYPE_ASE, 'Allele Specific Expression'),
+    DATASET_TYPE_READ_ALIGNMENTS = 'ALIGN'
+    DATASET_TYPE_VARIANT_CALLS = 'VARIANTS'
+    DATASET_TYPE_SV_CALLS = 'SV'
+    DATASET_TYPE_BREAKPOINTS = 'BREAK'
```

Comment:
what is a breakpoint?

Reply:
instead of fully calling structural variants (which includes 2 or more break-points + deciding if it's a deletion, insertion, inversion, etc.), you can go halfway and just detect the DNA breakpoints